File: simple/jack3.cs
using System;
using System.Collections.Generic;
using System.Linq;

class BlackJack{
	static int bet=200;
	static int getSum(List<int> card){
		var cardAceSum=card.Select(v=>v==1?11:v).Sum();
		return cardAceSum<=21?
			cardAceSum:
			card.Sum();
	}

	static void Main(){
		Func<List<int>> getCard=()=>new List<int>(Console.ReadLine().Split(' ').Select(int.Parse));
		var myCard=getCard();
		var turn=int.Parse(Console.ReadLine());
		var combo=int.Parse(Console.ReadLine());

		if(myCard[0]==0){
			Console.WriteLine(bet); // 賭けチップ数
		}
		else{
			var maxBet=int.Parse(Console.ReadLine());
			var cpuCard=getCard();

			if(getSum(myCard)<=getSum(cpuCard) || getSum(myCard)<14){ // ★カードを引く条件の合計値を変えてみよう！★
				Console.WriteLine("HIT"); // カード引く
			}
			else{
				Console.WriteLine("STAND"); // 勝負
			}
		}
	}
}

File: simple/jack3.py
bet=200
def getSum(card):
	cardAceSum=sum([11 if v==1 else v for v in card])
	return (
		cardAceSum if cardAceSum<=21 else 
		sum(card))

getCard=lambda:list(map(int,input().split(" ")))
myCard=getCard()
turn=int(input())
combo=int(input())

if myCard[0]==0:
	print(bet) # 賭けチップ数
else:
	maxBet=int(input())
	cpuCard=getCard()

	if getSum(myCard)<=getSum(cpuCard) or getSum(myCard)<14: # ★カードを引く条件の合計値を変えてみよう！★
		print("HIT") # カード引く
	else:
		print("STAND") # 勝負
